crates/service-protocol/service-protocol/dev/restate/service/protocol.proto

```diff
@@ -40,10 +40,8 @@ message StartMessage {
   repeated StateEntry state_map = 4;
   bool partial_state = 5;
 
-  repeated Header headers = 6;
-
   // If this invocation has a key associated (e.g. for objects and workflows), then this key is filled in. Empty otherwise.
-  string key = 7;
+  string key = 6;
```

Comment:
I assume this warrants a protocol version bump if we haven't done it already compared to version `0.8`.

Reply:
I was thinking to do this bump later, but ok will do it now.